docs/reference/data-streams/lifecycle/tutorial-manage-existing-data-stream.asciidoc

```diff
@@ -0,0 +1,136 @@
+[role="xpack"]
+[[tutorial-manage-existing-data-stream]]
+=== Tutorial: Update existing data stream
+
+preview::[]
+
+To update the lifecycle of an existing data stream you do the following actions:
+
+. <<set-lifecycle>>
+. <<delete-lifecycle>>
+
+[discrete]
+[[set-lifecycle]]
+==== Set a data stream's lifecycle
+
+To add or to change the retention period of your data stream you can use the <<data-streams-put-lifecycle, PUT lifecycle API>>.
+
+* You can set infinite retention period, meaning that your data should never be deleted. For example:
++
+[source,console]
+----
+PUT _data_stream/my-data-stream/_lifecycle
+{ } <1>
+----
+// TEST[setup:my_data_stream]
+<1> An empty payload means that your data stream is still managed but the data will never be deleted. Managing a time
+series data stream such as logs or metrics enables {es} to better store your data even if you do not use a retention period.
+
+* Or you can set the retention period of your choice. For example:
```

Comment:
Do we indicate somewhere what values are allowed for the data_retention? What is the minimum, maximum, and the format etc?

Reply:
Hm, no we haven't, not explicitly at least, it's in the examples. Do you think we should add this?